backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -119,12 +120,15 @@ private String dateToHealthCareString(LocalDate value) {
     return value.format(DateTimeFormatter.ofPattern("yyyyMMdd"));
   }
 
-  private String dateToHealthCareString(Optional<LocalDate> value) {
```

Comment:
Optional is really intended as a return type more than an argument. This method _performs_ a null check internally, so this original overload is kind of awkward. We also can't overload this function with `Optional<LocalDate>` + `Optional<LocalDateTime>` because they reduce to the same `Optional<Object>`

Anyway I've moved this `.orElse(null)` downstream accordingly, and it ends up being a bit cleaner because now the two signatures for `dateToHealthCareString` are with a single argument of either `LocalDate` and `LocalDateTime`, and serializes accordingly